include/fastrtps/utils/fixed_size_string.hpp

```diff
@@ -170,6 +170,13 @@ struct fixed_string
         return string_len;
     }
 
+    /*!
+     * Compare with a std:.string.
```

Comment:
Little typo: std::string instead of std:.string

Reply:
Done in a4d80e5